modular_zubbers/code/modules/status_indicators/status_indicator_pref.dm

```diff
@@ -1,26 +0,0 @@
-/datum/preference/toggle/enable_status_indicators
```

Comment:
if this file is empty it should be deleted and unticked from the dme not just left empty